hazelcast-wm/pom.xml

```diff
@@ -105,6 +105,11 @@
                                     org.apache.logging.log4j;resolution:=optional,
                                     org.apache.logging.log4j.*;resolution:=optional,
                                     org.slf4j;resolution:=optional,
+                                    org.springframework.context;resolution:=optional,
+                                    org.springframework.security.core.session;version="[3.2,4)";resolution:=optional,
```

Comment:
@fwassmer , Why do you provide version here? Is it intentional? What is the logic behind this?